app/models/krikri/validation_report.rb

```diff
@@ -0,0 +1,54 @@
+module Krikri
+  class ValidationReport
+    attr_accessor :provider_id, :start, :rows
+
+    REQUIRED_FIELDS = ['dataProvider_name', 'isShownAt_id', 'preview_id',
+                       'sourceResource_rights', 'sourceResource_title',
+                       'sourceResource_type_id']
+
+    ##
+    # @param &block may contain provider_id
+    #   Sample use:
+    #     ValidationReport.new.all do
+    #       self.provider_id = '0123'
+    #     end
+    #
+    # @return [Array<Blacklight::SolrResponse::Facet>]
+    def all(&block)
+      # set values from block
+      instance_eval &block if block_given?
+
+      query_params = { :rows => 0,
+                       'facet.field' => REQUIRED_FIELDS,
+                       'facet.mincount' => 10000000,
+                       'facet.missing' => true }
+      query_params[:fq] = "provider_id:\"#{@provider_id}\"" if
+        @provider_id.present?
```

Comment:
When I tested this in my browser, I noticed that reports do not seem to be correctly filtering by provider.  When I load `http://localhost:3000/krikri/validation_reports/sourceResource_title`, I see one record :+1:  When I load `http://localhost:3000/krikri/validation_reports/sourceResource_title?provider=123`, I see no records, but I think I should see one, correct?

I think the problem is that `@provider_id` is a local name, and the solr query param:
```query_params[:fq] = "provider_id:\"#{@provider_id}\""``` 
is trying to make and exact match between the indexed `provider_id` and the local name.  

Maybe try something like:
```provider_uri = RDF::URI(Krikri::Provider.base_uri) / @provider_id```
```query_params[:fq] = "provider_id:\"#{provider_uri}\""```